cms/envs/common.py

```diff
@@ -20,19 +20,19 @@
    multiple sites, but we do need a way to map their data assets.
 
 When refering to XBlocks, we use the entry-point name. For example,
```

Comment:
You need to end this line with two colons, and leave a blank line after.  Thanks for your conscientious approach :)
```
For example::

    setup(
        ...
```